openssh/Patches/0022-Add-SMF-manifest-and-method-and-install-them.patch

```diff
@@ -266,16 +265,6 @@ index 00000000..e91ed553
 +	fi
 +}
 +
```

Comment:
This probably needs to go back to how it was

Reply:
Looks like it was  changed when the smartos-live overlay was merged in in 45e1cee06836861ef79ebe2514a19c956ee96b92, but the illumos-openssh patch set wasn't updated